includes/deploy/class-menus.php

```diff
@@ -90,7 +90,7 @@ public function deploy() {
 
 		foreach ( $configs['menus'] as $term_id => $menu_data ) {
 			// Create the menu.
-			$menu_id = wp_create_nav_menu( $menu_data['name'] . '-test' );
```

Comment:
I think you added it so you could verify that the menu was being created during testing. This is being removed because we don't actually want '-test' added to the menu name.